src/services/daemon.service.ts

```diff
@@ -105,6 +110,7 @@ export class DaemonService implements iDaemonService {
 
     this.rskBlockProcessorPublisher.addSubscriber(this.peginDataProcessor);
     this.rskBlockProcessorPublisher.addSubscriber(this.pegoutDataProcessor);
+    this.rskBlockProcessorPublisher.addSubscriber(this.pegnatoriesDataProcessor);
```

Comment:
Don't forget to remove the subscriber on method stop()

`this.rskBlockProcessorPublisher.removeSubscriber(this.pegnatoriesDataProcessor);`

Reply:
Right, we overlooked it. Fixed now